packages/ni.measurementlink.sessionmanagement.v1.client/src/ni/measurementlink/sessionmanagement/v1/client/_types.py

```diff
@@ -139,6 +139,13 @@ class SessionInformation(NamedTuple):
     This field is None until the appropriate initialize_session(s) method is called.
     """
 
+    annotations: dict[str, str] = {}
```

Comment:
FYI: this breaks compatibility with code that unpacks the named tuple into separate variables, like `x, y, z = mytuple`. However, we have done this before. No action needed.